xsync/Cargo.toml

```diff
@@ -5,6 +5,9 @@ default-members = ["xsync"]
 members = ["xsync"]
 resolver = "2"
 
+[workspace.package]
+rust-version = "1.82"
```

Comment:
This will need to be kept in sync with the root manually? Or maybe we can just let it be old if xsync doesn't need anything new?

Reply:
I think it'll be fine if this has its own MSRV.

Even more broadly: I don't really expect `xsync` to be something that gets updated all that often. Heck, in many ways, I think this entire tool should be in its on standalone repo, and rather than us compiling it from scratch each time, we pull down a precompiled copy of it.